modules/1_prepare/variables.tf

```diff
@@ -19,7 +19,12 @@
 ################################################################
 
 variable "cluster_domain" {}
-variable "cluster_id" {}
+variable "cluster_id" {
+    validation {
```

Comment:
Could you please add `type` here.

Reply:
The cluster_id length check is needed on prepare variables.tf?

Reply:
Since you have it, lets keep it.

Reply:
Ok, I have added type constraint.